Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG UBI_IMAGE=registry.access.redhat.com/ubi7/ubi-minimal:latest
-ARG GO_IMAGE=rancher/hardened-build-base:v1.16.10b7
+ARG UBI_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
I am not sure we can move to ubi8 since the FIPs compliant certificate we got is based on ubi7. This might void that certificate. I need to check

Reply:
I can gladly role this one back :+1:

Reply:
And leave this in there: https://github.com/rancher/image-build-flannel/pull/8